SETUP/tests/jsTests/formatPreviewTests.js

```diff
@@ -97,8 +97,8 @@ QUnit.module("Format preview test", function() {
     QUnit.test("unrecognised tag, u not enabled", function (assert) {
         text = "ab <u>cd</u>";
         issArray = analyse(text, configuration);
-        issueTest(assert, 1, 3, 1, "unRecTag", 0);
-        issueTest(assert, 0, 8, 1, "unRecTag", 0);
+        issueTest(assert, 1, 3, 3, "unRecTag", 0);
```

Comment:
why did this change? I don't see the test being related to math at all.

Reply:
I'm sure there was a good reason. Several commits were merged together several months ago for this MR because of a problem with conflicts with some other changes which were happening at the same time so the original issue has got lost. Perhaps I should redo it all.